DoublyLinkedList/doubly_linked_list.h

```diff
@@ -0,0 +1,213 @@
+#include <iostream>
+using namespace std;
+// Node Creation
+class doubly_linked_list_node
+{
+    public:
+    int data;
+    doubly_linked_list_node *next;
+    doubly_linked_list_node *prev;
+};
+
+class doubly_linked_list
+{ //class
+        doubly_linked_list_node *head;
+
+
+public:      
+        doubly_linked_list(){
+            head=NULL;
+        };                                                //Access specifier
+        void push(int value); // insertion of value at front
+
+        void append(int value); //insertion of value at end
+
+        void insert(int value, int location); //for inserting at location
+
+        void delete_node(int value); //for deleting a node
```

Comment:
these delete apis are not useful. we can remove them

Reply:
ok sir